app/data/1_0001.json

```diff
@@ -4859,7 +4890,12 @@
                                             "mandatory": false,
                                             "options": [],
                                             "q_code": "2620",
-                                            "type": "Percentage"
+                                            "type": "Percentage",
+                                            "validation": {
+                                                "messages" : {
+                                                    "INTEGER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
```

Comment:
It is defined as a global message, it's being overridden for Percentage fields so that the value reads 100 instead of 9999999999. I'm not sure if it's possible to define messages at a Question level?